lizmap/modules/lizmap/lib/App/AppContextInterface.php

```diff
@@ -65,14 +65,25 @@ public function aclCheck($role, $resource = null);
     public function aclUserGroupsId();
 
     /**
-     * Returns the groups ids of an user.
+     * Retrieve the list of group the given user is member of
+     * in the acl system.
      *
-     * @param string $login The login of the user
+     * @param string $login The user's login
      *
-     * @return array an array containing the user's groups ids
+     * @return array list of group id
      */
     public function aclGroupsIdByUser($login);
 
+    /**
+     * Get the private group for the current user or for the given login
+     * in the acl system.
+     *
+     * @param string $login The user's login
+     *
+     * @return string the id of the private group
+     */
+    public function aclUserPrivateGroup($login = null);
```

Comment:
It would be better to have a new method `aclPublicGroupsIdByUser($login)`. It would avoid to repeat the code that remove the private group from the list of groups given by `aclGroupsIdByUser`.

And this new method could call `aclGroupsIdByUser()`, and just remove id which begins with `"__priv__"`, like [we do in QgisForm:: updateFormByLogin()](https://github.com/3liz/lizmap-web-client/blob/master/lizmap/modules/lizmap/lib/Form/QgisForm.php#L1108) .